tests/support/util.tcl

```diff
@@ -571,7 +571,7 @@ proc wait_load_handlers_disconnected {{level 0}} {
     }
 }
 
-proc K { x y } { set x } 
+proc K { x y } { set x }
```

Comment:
Avoid unnecessary touch.

Reply:
@hpatro I thought it would be nice to remove the unnecessary blank space, but I've removed the unnecessary changes as you suggested.

Thanks for the feedback